server/routes/validation/auth.js

```diff
@@ -4,7 +4,9 @@ const User = require("../../models/User");
 const validation = [
   body("name")
     .trim()
-    .isEmpty(),
+    .custom((value, { req }) => {
+      return value.length > 0;
+    }, 'name field must not be empty'),
```

Comment:
this custom error never shows up, only getting 
{
    "message": "validation failed",
    "errorData": [
        {
            "value": "",
            "msg": "Invalid value",
            "param": "name",
            "location": "body"
        }
    ]
}